### ISSUE TITLE
👌 IMPROVE: Structure selection

### ISSUE THREAD

**Issue by mbercx** [aiidalab/aiidalab-widgets-base#250]
Several small comments from @giovannipizzi on the Structure selection step:

- Was the table with atomic coordinates removed? (I mean the one near the 3D visualisation, before submission) (#126)
- The POVRAY renderer does not seem to take into account the supercell settings. (#127)
- When rendering, is it possible to run in the background, so it does not block the interaction with the rest of the notebook? (or warn with an "alert" that this will take some seconds and block?)
- Every time I press the screenshot or Render button, I get a message from my browser (Safari) that the page is going to change (see attachment screenshot1). Do you know why?
- In Safari, the download screenshot downloads a file called "Unknown" without extension, so it's not obvious it's a PNG
- I would add at least the XSF format for the download
- one should clarify if the supercell is just for visualisation or also for the simulation (I had this doubt)

**Comment by yakutovicha** [comment]
> * Was the table with atomic coordinates removed?

I wonder, is it needed at all?

**Comment by csadorf** [comment]
> > * Was the table with atomic coordinates removed?
> 
> I wonder, is it needed at all?

Yes, this was also recorded in #249 or am I misunderstanding something?

**Comment by yakutovicha** [comment]
> > > * Was the table with atomic coordinates removed?
> > 
> > 
> > I wonder, is it needed at all?
> 
> Yes, this was also reported in #249 or am I misunderstanding something?

I think #249 is about cell parameters, while here it refers to the atomic coordinates.

**Comment by giovannipizzi** [comment]
Sometimes you might just want to check the coordinates so seeing them in an easy way might be useful. Good to check with more users if they feel this is useful, if more want it, I guess it's useful, otherwise it's just me (and then I can download as XSF and check, or similar)

**Comment by yakutovicha** [comment]
> * [ ]  Every time I press the screenshot or Render button, I get a message from my browser (Safari) that the page is going to change (see attachment screenshot1). Do you know why?
> * [ ]  In Safari, the download screenshot downloads a file called "Unknown" without extension, so it's not obvious it's a PNG

Non of these two I can reproduce. @giovannipizzi, can you please try again?

**Comment by giovannipizzi** [comment]
> In Safari, the download screenshot downloads a file called "Unknown" without extension, so it's not obvious it's a PNG

This I cannot reduce anymore

> Every time I press the screenshot or Render button, I get a message from my browser (Safari) that the page is going to change (see attachment screenshot1). 

This is still there, for the Screenshot button only (render, instead, works), see here:

![Screenshot 2021-10-05 at 09 22 58](https://user-images.githubusercontent.com/16132028/135978310-f05b61b5-826e-4954-abd5-eda1ce43e0e2.png)

I'm not sure why - I think it is for some reason telling the browser to change the page while it's stil; processing, maybe?
I get the same message sometimes e.g. when I close the home page (or between switching pages).
